pyproject.toml

```diff
@@ -73,18 +73,18 @@ ci =  "https://github.com/ansys/pydynamicreporting/actions"
 tests = [
     "docker>=7.1.0",
     "numpy==1.25.1",
-    "psutil==5.9.5",
-    "pytest==7.4.2",
+    "psutil==6.0.0",

```

Comment:
@margalva mental note here.   We have to be careful with this package and integrated pytest execution (like we do with pyensight) as these pins would cause problems if someone tried to push deps before running pytests within an ADO build.